### ISSUE TITLE
Container cookies not being deleted

### ISSUE THREAD

**Issue by m4teh** [mrdokenny/Cookie-AutoDelete#256]
Cookies generated within a container don't seem to be cleared.
Tested and reproduced on a fresh Firefox 57.0 install with default browser configuration in a Windows 7 VM (in case of conflicting addon etc)

[tick] Enable Automatic Cleaning? Delay Before Cleaning:Minute(s)?
[tick] Log Total Number Of Cookies Deleted?
[tick] Show Number of Cookies for that Domain?
[tick] Show Notification After Cookie Cleanup?
Clean Cookies from Open Tabs on Startup?
[tick] Enable Support for Firefox's Container Tabs?

- OS/version: Windows 10 64-bit
- Browser/version: 57.0 64-bit
- Cookie AutoDelete version: 2.0.1

**Comment by cherti** [comment]
Can confirm for Linux as well.

OS/Version: Arch Linux
Browser/Version: Firefox 57.0
CAD version: 2.0.1

**Comment by cincinnatus0** [comment]
Just noticed the same issue as @m4teh is facing on FF 57.0 W10 64bit. Hope @mrdokenny you can fix it soon. Thank you in advance!

**Comment by practik** [comment]
Same problem here too:

OS version: macOS 10.12.6
Browser version: Firefox 57.0
CAD version: 2.0.1

**Comment by ghost** [comment]
https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/issues/197#issuecomment-337523756

**Comment by practik** [comment]
@gwarser, thanks for the workaround, it's working for me.  I hadn't even noticed that the CAD icon was blue on my container tab, but after creating and then deleting a whitelist rule in the tab as you suggested, I found that the icon turned red and the tab's cookies were deleted after I closed it. I don't like the way it interacts with open tabs in other containers, but I've opened a new issue for that at #262.
